PhoneBook.Console/Program.cs

```diff
@@ -1,13 +1,14 @@
-﻿using System.Linq;
+﻿using System.Collections.Generic;
+using System.Linq;
 using PhoneBook.Domain;
 using Remotion.Data.DomainObjects;
 using Remotion.Data.DomainObjects.Queries;
 
 namespace PhoneBook.Console
 {
-  class Program
+  internal static class Program
   {
-    public static Person[] GetPersons ()
+    private static IEnumerable<Person> GetPeople ()
```

Comment:
The reasoning behind "Persons" is that it is the plural of "Person", the domain type for which the query result is returned. We want to keep this naming convention.